backend/models/modelstesting/testhelpers.go

```diff
@@ -0,0 +1,28 @@
+package modelstesting
+
+import (
+	"socialpredict/handlers/math/probabilities/wpam"
+	"socialpredict/models"
+	"time"
+)
+
+// GenerateBet is used for Generating fake bets for testing purposes
+// helper function to generate bets succinctly for testing
+func GenerateBet(amount int64, outcome, username string, marketID uint, offset time.Duration) models.Bet {
+	return models.Bet{
+		Amount:   amount,
+		Outcome:  outcome,
+		Username: username,
+		PlacedAt: time.Now().Add(offset),
+		MarketID: marketID,
+	}
+}
+
+// helper function to create wpam.ProbabilityChange points succinctly
+func GenerateProbability(probabilities ...float64) []wpam.ProbabilityChange {
+	var changes []wpam.ProbabilityChange
+	for _, p := range probabilities {
+		changes = append(changes, wpam.ProbabilityChange{Probability: p})
+	}
+	return changes
+}
```

Comment:
This function is not related to the models in any way. It should be added to a `wpam/wpamtesting` module that can be used anywhere the `wpam` module is already available.